web/app.js

```diff
@@ -41,6 +41,12 @@ import { SecondaryToolbar } from 'pdfjs-web/secondary_toolbar';
 import { Toolbar } from 'pdfjs-web/toolbar';
 import { ViewHistory } from 'pdfjs-web/view_history';
 
+var {
```

Comment:
Good call. I didn't notice that, so I wondered why this was different. Looks good to me in that case.